index.js

```diff
@@ -124,7 +131,9 @@ module.exports = {
     'object-curly-spacing': ['error', 'never'],
     'no-nested-ternary': 'warn',
     'multiline-comment-style': 'warn',
-    "spaced-comment": ["warn", "always", { "exceptions": ["-", "+"] }]
+    "spaced-comment": ["warn", "always", { "exceptions": ["-", "+"] }],
+    'no-console': 'warn',
+    'default-param-last': 'warn',
```

Comment:
default param should be in last position. [link](https://eslint.org/docs/latest/rules/default-param-last)